packages/flutter/test/material/date_picker_test.dart

```diff
@@ -622,4 +622,77 @@ void _tests() {
 
     semantics.dispose();
   });
+
+
+  testWidgets('chervons animate when scrolling month picker', (WidgetTester tester) async {
+    final Key _datePickerKey = new UniqueKey();
+    DateTime _selectedDate = new DateTime(2016, DateTime.july, 26);
+
+    await tester.pumpWidget(
+      new MaterialApp(
+        home: new StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return new Container(
+              width: 400.0,
+              child: new SingleChildScrollView(
+                child: new Material(
+                  child: new MonthPicker(
+                    firstDate: new DateTime(0),
+                    lastDate: new DateTime(9999),
+                    key: _datePickerKey,
+                    selectedDate: _selectedDate,
+                    onChanged: (DateTime value) {
+                      setState(() {
+                        _selectedDate = value;
+                      });
+                    },
+                  ),
+                ),
+              ),
+            );
+          },
+        ),
+      )
+    );
+
+
+    for(Element element in find.byType(IconButton).evaluate()) {
```

Comment:
https://docs.flutter.io/flutter/flutter_test/CommonFinders/descendant.html

You can define a finder value once rather than using this block of code over and over.

Reply:
Switching to:

final Finder chevronFinder = find.byType(IconButton);
chevronFinder.precache();